insights/parsers/abrt_status_bare.py

```diff
@@ -0,0 +1,25 @@
+"""
+AbrtStatusBare - command ``/usr/bin/abrt status --bare=True``
+=============================================================
+
+``/usr/bin/abrt status --bare=True`` returns the number of problems ABRT
+detected in the system.
+
+Examples:
+    >>> status.problem_count
+    0
+"""
+
+from insights import CommandParser, parser
+from insights.specs import Specs
+
```

Comment:
You have flake8 errors here, you can see this in the CI results.  You can test this by running `flake8` in your project, and test by running `pytests`.  Documentation generation can be tested by following the documentation [here](https://insights-core.readthedocs.io/en/latest/docs_guidelines.html#testing-your-docstring).